daemon/dpudaemon.go

```diff
@@ -17,16 +17,17 @@ type DpuDaemon struct {
 	pb.UnimplementedBridgePortServiceServer
 	vsp plugin.VendorPlugin
 	log logr.Logger
+	server *grpc.Server
 }
 
 func (s *DpuDaemon) CreateBridgePort(context context.Context, bpr *pb.CreateBridgePortRequest) (*pb.BridgePort, error) {
-	s.log.Info("Create Bridge Port", "req", bpr.BridgePort.Name)
-	return &pb.BridgePort{}, nil
+	err := s.vsp.CreateBridgePort(bpr)
+	return nil, err
```

Comment:
why ``nil, err``?

Reply:
I have the same question. I have the same question, why are no longer returning an empty bridge port and instead of nil and err?  If we are only going to return nil, it would make sense to return only err.